src/supporting-data/database/config/download.ts

```diff
@@ -2,5 +2,4 @@ import { FileFormat } from '../../../shared/types/resultsDownload';
 
 export const fileFormatsResultsDownload = [FileFormat.rdfXml, FileFormat.json];
```

Comment:
Noticed that the "Preview N" button in download doesn't work for RDF/XML - (error in the request syntax?)

Reply:
It's the issue I mentioned this morning. RDF/XML is not supported by the backend (at the moment). I have now removed it.